keystorage/aws/src/main/java/tech/pegasys/signers/aws/SecretValueConverterUtil.java

```diff
@@ -26,27 +26,22 @@ public class SecretValueConverterUtil {
 
   public static <R> Set<R> mapSecretValue(
       BiFunction<String, String, R> mapper, String propKey, String secretValue) {
-    final AtomicLong discardedValues = new AtomicLong(0);
+    final AtomicLong valuesIndex = new AtomicLong(0);
     final Set<R> mappedValues =
         secretValue
             .lines()
             .map(
                 v -> {
+                  long index = valuesIndex.getAndIncrement();
                   final R obj = mapper.apply(propKey, v);
                   if (obj == null) {
-                    discardedValues.incrementAndGet();
+                    LOG.warn("Secret Value {}:{} was not mapped and discarded.", propKey, index);
```

Comment:
I would read "Secret Value" as the secret value.

Suggest "Secret key {} with index {} ..."?

Reply:
Does secret key mean anything to the users, from a user's point of view, what actually is the secret key with this multi-secret setup?

Reply:
"multi-part key"?

Reply:
Now that this code is in common module, the verbage has to be common enough to work for AWS and Azure etc. `Secret key {} with index {}` ... make sense to me.